persistence/src/main/java/com/_3po_labs/derpwizard/persistence/configuration/UserPreferencesDAOConfig.java

```diff
@@ -0,0 +1,58 @@
+package com._3po_labs.derpwizard.persistence.configuration;
```

Comment:
Take this class back to the default tab spacing. Now's a good time to restore consistency